pymc/step_methods/metropolis.py

```diff
@@ -140,22 +140,22 @@ def __init__(
 
         Parameters
         ----------
-        vars: list
+        vars : list
             List of value variables for sampler
-        S: standard deviation or covariance matrix
+        S : standard deviation or covariance matrix
             Some measure of variance to parameterize proposal distribution
-        proposal_dist: function
+        proposal_dist : function
             Function that returns zero-mean deviates when parameterized with
             S (and n). Defaults to normal.
-        scaling: scalar or array
+        scaling : scalar or array
```

Comment:
```suggestion
        scaling : scalar or array_like, default 1
```

for inputs unless it is required they are already a numpy array it is better to use array_like instead.

Reply:
I have changed this one based on your suggestions.